.changeset/metal-hornets-travel.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluidframework/container-runtime": patch
+---
+---
+"section": "fix"
+---
+Restored old op processing behavior around batched ops to avoid potential regression
+
+There's a theoretical risk of indeterminate behavior due to a recent change to how batches of ops are processed.
```

Comment:
```suggestion
Due to a recent change to how batches of ops are processed, there's a potential risk of indeterminate sequence numbers between layers.
```
My suggestion may not have the technical details 100% correct, but I think similar wording would be more clear.

Reply:
As discussed - Won't Fix (to match 2.3 release notes)